install_config/upgrading/manual_upgrades.adoc

```diff
@@ -215,7 +280,6 @@ endif::[]
 +
 ----
 # oadm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
     --exclude-groups=system:authenticated:oauth \
```

Comment:
No, the more correct change would be to tweak the `reconcile-cluster-roles` invocation above to add `--additive-only=true`, which would have prevented removing the source build permissions from the old admin/edit roles during an upgrade. 

`--additive-only=true` is now the default, but that may have changed after 3.2 was cut